stdlib/contrib/RohanSreerama5/naiveBayesClassifier/naiveBayesClassifier.flux

```diff
@@ -0,0 +1,114 @@
+//Final working code as of August 11, 2020
+//Currently supports single field classification and binary data sets 
+
+package naiveBayesClassifier
+
+import "system"
+
+
+
+////////csv.from(csv: naiveBayesClassifier/zoo_data.csv) |> naiveBayes()
+
+//---
```

Comment:
I know this isn't the final version, but just a reminder to go through and remove all the commented out lines of Flux.